homeassistant/components/huawei_lte/config_flow.py

```diff
@@ -119,11 +127,19 @@ async def _connect(
         password = user_input.get(CONF_PASSWORD) or ""
 
         def _get_connection() -> Connection:
+            if user_input[CONF_URL].startswith("https://") and not user_input.get(
```

Comment:
`CONF_VERIFY_SSL` has a default value, why do we use `get` here?

Reply:
d25b22448e818c4b8951fff1ac7d479967b0e3a8